core/all/src/main/scala/org/typelevel/otel4s/Otel4s.scala

```diff
@@ -20,7 +20,7 @@ import org.typelevel.otel4s.metrics.MeterProvider
 
 trait Otel4s[F[_]] {
 
-  /** A registry for creating named [[org.typelevel.otel4s.metrics.Meter]].
+  /** A registry for creating named meters.
```

Comment:
I think you can use a pipe operator if you want it to say `meters` but link to `Meter`:

```scala
[[org.typelevel.otel4s.metrics.Meter|meters]]
```

That should not slow this down.

Reply:
`doc` command was failing. I do not remember the exact reason, but I will bring back the link if it's possible.

Reply:
@iRevive you should just use a space there, not a `|`.